resqpy/olio/xml_et.py

```diff
@@ -713,14 +696,14 @@ def _escaped_text(text):
     attrib_ns_list = []
     attrib_list = []
     type_pre_colon = None
-    for key in root.attrib.keys():
+    for key, val in root.attrib.items():
         colon_attrib_key, pre_colon_attrib = colon_prefixed(key)
         if pre_colon_attrib and pre_colon_attrib not in ns_keys:
             attrib_ns_list.append(pre_colon_attrib)
         if match(key, 'type'):
-            type_attr, type_pre_colon = colon_prefixed(root.attrib[key])
+            type_attr, type_pre_colon = colon_prefixed(val)
             attrib_list.append(colon_attrib_key + '="' + type_attr + '"')
-        elif ct_special and colon_attrib_key == 'PartName' and root.attrib[key].startswith('obj_'):
+        elif ct_special and colon_attrib_key == 'PartName' and val.startswith('obj_'):
             attrib_list.append(colon_attrib_key + '="/' + root.attrib[key] + '"')
```

Comment:
I believe this _root.attrib[key]_ can be replaced by _val_ (also a couple of lines below)

Reply:
Yeah that was a change I made, then reverted back by accident